le_dash/es.py

```diff
@@ -25,7 +27,15 @@ def to_dict(self):
         return self.search.to_dict()
 
     def execute(self):
-        resp = self.search.execute()
+        key = hashlib.md5("{0}".format(self.to_dict()))
+        try:  # Do not crash if cache is missing
+            resp = cache.get(key)
+            if not resp:
+                resp = self.search.execute()
+                if resp.hits > 0:   # only cache good results
+                    cache.set(key, resp)
```

Comment:
I don't see how this could work. The `resp` value is an instance of `ESResponse`. To cache it in redis it would need to be serialized in some way, then recreated in some manner.